go/libkb/merkle_client.go

```diff
@@ -394,6 +395,34 @@ func GetNodeHashVoid(w *jsonw.Wrapper, nhp *NodeHash, errp *error) {
 	}
 }
 
+func computeSetBitsBigEndian(x int) []int {
+	var ret []int
+	bit := 1
+	for i := 0; i <= int(math.Ceil(math.Log2(float64(x)))); i++ {
```

Comment:
pull this (floating point) out of the loop so it's only performed once, not every time through the loop.  And plus, would we prefer a discrete log2 that just uses bitshifting? (Or maybe something crazy like this: https://stackoverflow.com/questions/11376288/fast-computing-of-log2-for-64-bit-integers)

Reply:
Use `uints` or `uint64s`?